app/uk/gov/hmrc/homeofficeimmigrationstatus/views/StatusFoundPageContext.scala

```diff
@@ -50,6 +50,26 @@ final case class StatusFoundPageContext(
       case None => messages(prefix + "noStatus")
     }
   }
+
+  def getImmigrationStatus(productType: String) =
+    Map(
+      "EUS"             -> "EU Settlement Scheme",
+      "STUDY"           -> "Student (FBIS)",
+      "DEPENDANT"       -> "Dependants of Skilled workers and Students (FBIS)",
+      "WORK"            -> "Worker (FBIS)",
+      "FRONTIER_WORKER" -> "Frontier worker (FBIS)",
+      "BNO"             -> "British National Overseas (FBIS)",
+      "BNO_LOTR"        -> "British National Overseas (FBIS)",
+      "GRADUATE"        -> "Graduate (FBIS)",
+      "SPORTSPERSON"    -> "International Sportsperson (FBIS)",
+      "SETTLEMENT"      -> "British National Overseas (FBIS)",
+      "TEMP_WORKER"     -> "Temporary Worker (FBIS)"
+    ).withDefaultValue("Unknown Immigratiomn Route")(productType)
```

Comment:
is this the agreed default or just a place holder? id imagine we still want to show whatever it was to give the case worked a chance maybe : 
```suggestion
    ).withDefault(identity)(productType)
```

Reply:
Yeah, we want to default it back to the product type code really. I think a separate ticket will come in for "future proofing" which will cover a lot of these, but we know the functionality will be something like this